playground/jest.config.ts

```diff
@@ -1,7 +1,7 @@
 module.exports = {
```

Comment:
This is only made prettier using `npx prettier --write .`. I wasn't able to undo that step. I can manually remove those files from the PR if you want.